dcrjson/dcrwalletextcmds.go

```diff
@@ -294,6 +294,49 @@ func NewPurchaseTicketCmd(fromAccount string, spendLimit float64, minConf *int,
 	}
 }
 
+// StartAutoBuyerCmd is a type handling custom marshaling and
+// unmarshaling of startautobuyer JSON RPC commands.
+type StartAutoBuyerCmd struct {
+	Passphrase        *[]byte
```

Comment:
I don't believe all parameters are intended to be optional, are they?  A pointer for command struct fields denotes an optional parameter in the `dcrjson` code.